lib/structures/Message.js

```diff
@@ -333,6 +333,15 @@ class Message extends Base {
         return this._client.removeMessageReactions.call(this._client, this.channel.id, this.id);
     }
 
+    /**
+    * Remove all reactions from a message for a single emoji
+    * @arg {String} reaction The reaction (Unicode string if Unicode emoji, `emojiName:emojiID` if custom emoji)
+    * @returns {Promise}
+    */
+    removeReactionsForEmoji(reaction) {
```

Comment:
This method name should match what's present in the client.

Reply:
@Brayzure This is the same name following the consistency in the other methods. The other methods also remove `Message` from their names so i had kept it that way as well.

Reply:
Oh true, I do see that.